openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -361,7 +361,14 @@ def login_user(request):
             _handle_failed_authentication(user, possibly_authenticated_user)
 
         if not is_edly_user_allowed_to_login(request, possibly_authenticated_user):
-            raise AuthFailedError(_('You are not allowed to login on this site.'))
+            # Check if user registered to any sibling sub-organization of this sub-organization
```

Comment:
nit: If we name or variables and methods properly and use clean code practices then there is no need to add comments to describe our code. Please remove this comment.